source/includes/azure/_instances.md

```diff
@@ -250,6 +250,42 @@ Required | &nbsp;
 ------ | -----------
 `machineType`<br/>*string* | The new machine type to assign to the instance
 
+<!-------------------- RESET PASSWORD -------------------->
+
+#### Reset password
+
+The admin password is used to access Linux or Windows [instances](#azure-instances). 
+
+```shell
```

Comment:
It should say :
Reset the administrator password on a Windows or Linux [instances] (#azure-instances).

Reply:
Done!